modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/display/context/FDSSampleDisplayContext.java

```diff
@@ -61,6 +69,69 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems()
 		String portalURL = company.getPortalURL(
 			GroupConstants.DEFAULT_PARENT_GROUP_ID);
 
+		FDSActionDropdownItem openEmptySidePanelAction =
+			new FDSActionDropdownItem(
+				PortletURLBuilder.createRenderURL(
+					_renderResponse
+				).setMVCRenderCommandName(
+					"/side_panel/empty"
+				).setWindowState(
+					LiferayWindowState.POP_UP
+				).buildRenderURL(
+				).toString(),
+				"rectangle-split", "open-side-panel-no-title",
+				"Side Panel With Action Title", null, null, "sidePanel");
+
+		openEmptySidePanelAction.putData("disableHeader", false);
+		openEmptySidePanelAction.putData(
+			"title", "Side panel title provided by action");
+
+		FDSActionDropdownItem openFullSidePanelAction =
+			new FDSActionDropdownItem(
+				PortletURLBuilder.createRenderURL(
+					_renderResponse
+				).setMVCRenderCommandName(
+					"/side_panel/full"
+				).setWindowState(
+					LiferayWindowState.POP_UP
+				).buildRenderURL(
+				).toString(),
+				"rectangle-split", "open-side-panel-title",
+				"Side Panel With Content Title", null, null, "sidePanel");
+
+		openFullSidePanelAction.putData("disableHeader", true);
+
+		FDSActionDropdownItem openFullSidePanelActionDuplicatedTitle =
+			new FDSActionDropdownItem(
+				PortletURLBuilder.createRenderURL(
+					_renderResponse
+				).setMVCRenderCommandName(
+					"/side_panel/full"
+				).setWindowState(
+					LiferayWindowState.POP_UP
+				).buildRenderURL(
+				).toString(),
+				"rectangle-split", "open-side-panel-title",
+				"Side Panel With Action & Content Title", null, null,
+				"sidePanel");
+
+		openFullSidePanelActionDuplicatedTitle.putData("disableHeader", false);
+		openFullSidePanelActionDuplicatedTitle.putData(
+			"title", "Side panel title provided by action");
+
+		FDSActionDropdownItem openEmptySidePanelActionNoTitle =
+			new FDSActionDropdownItem(
+				PortletURLBuilder.createRenderURL(
+					_renderResponse
+				).setMVCRenderCommandName(
+					"/side_panel/empty"
```

Comment:
Nice, this is fully "by-the-book" :+1: 

A simpler but uglier way to do this would be to use `.setMVCPath("/path/to/empty.jsp")`